Source/Particles/Collision/MCCProcess.cpp

```diff
@@ -99,7 +100,8 @@ MCCProcess::readCrossSectionFile (
     std::ifstream infile(cross_section_file);
     if(!infile.is_open()) amrex::Abort("Failed to open cross-section data file");
 
-    double energy, sigma;
+    /* NOTE Were these declared double intentionally? */
```

Comment:
@roelof-groenewald is the change to amrex::Real OK? Is there a reason that this was declared double?

Reply:
The only reason for `double` is in case the cross-section related calculations benefit from the higher precision, but again, since the CI tests pass it is fine to use floats.